src/OpenFlight/Books/Domain/BookRepository.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+
+namespace CodelyTv\OpenFlight\Books\Domain;
+
+interface BookRepository
+{
+    public function create(Book $book): void;
```

Comment:
nitpick: El repositorio guarda, no crea
```suggestion
    public function save(Book $book): void;
```